saleor/graphql/order/mutations/fulfillments.py

```diff
@@ -2,10 +2,14 @@
 from django.core.exceptions import ValidationError
 from django.utils.translation import npgettext_lazy, pgettext_lazy
 
-from ....order import events, models
-from ....order.emails import send_fulfillment_confirmation_to_customer
+from ....order import models
+from ....order.actions import (
+    cancel_fulfillment,
+    fulfillment_tracking_updated,
+    order_fulfilled,
+)
 from ....order.error_codes import OrderErrorCode
-from ....order.utils import cancel_fulfillment, fulfill_order_line, update_order_status
+from ....order.utils import fulfill_order_line
```

Comment:
Isn't `fulfill_order_line` an *action* as well?

Reply:
I'm not sure what are the criteria for an *action* now, in other words, how do we decide whether a function should go to `actions.py`. From the function name `fulfill_order_line`, it sounds as an action.